ambry-api/src/main/avro/StatsDirectory.avsc

```diff
@@ -0,0 +1,9 @@
+{
+  "namespace": "com.github.ambry.store",
+  "type": "record",
+  "name": "StatsDirectory",
+  "fields": [
+    {"name": "value", "type": "long"},
```

Comment:
Sorry to bring this up now. But do you think we need to have a timestamp here. Basically incase a store wasn't available (or a disk), we would still update the stats for every other store. But the result should contain the old value for those unavailable stores. And this calls for a timestamp as to refer to when was it updated. Or to the least, we need the store Info.